### PR TITLE
AV-76530 extract capella docs

### PR DESCRIPTION
**Pull request by osfameron**
To point build at new docs-capella repo once
https://github.com/couchbasecloud/couchbase-cloud/pull/30453 is merged.

### CODE REVIEW

File: antora-playbook-staging.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ content:
     branches: [release/7.6, release/7.2, release/7.1, release/7.0, release/6.6, capella, elixir]
     start_path: docs
   - url: https://github.com/couchbasecloud/couchbase-cloud
```
Comment (RichardSmedley) [inline antora-playbook-staging.yml:56]: Doesn't the staging playbook need to update to   - url: https://github.com/couchbase/docs-capella as well?

Comment (osfameron) [reply antora-playbook-staging.yml:56]: It's already there from an earlier commit (I've built Staging to test with this configuration)

**Review by sarahlwelton — APPROVED** [review]
I see no issues. Approved.

**Review by dgnicholson — APPROVED** [review]
LGTM

**Pull request merged**